tests/kernel/interrupt/src/nested_irq.c

```diff
@@ -8,6 +8,18 @@
 #include <ztest.h>
 #include "interrupt_util.h"
 
+/*
+ * Run the nested interrupt test for the supported platforms only.
+ *
+ * NOTE: Cortex-A and Cortex-R platforms with the ARM GIC are skipped because
+ *       the arch port for these architectures do not support interrupt
+ *       nesting yet.
+ */
+#if defined(CONFIG_CPU_CORTEX_M) || defined(CONFIG_CPU_ARCV2) || \
+	(defined(CONFIG_GIC) && !defined(CONFIG_CPU_CORTEX_R))
```

Comment:
This does not look that CORTEX_A is skipped, right?

Reply:
Correct.

Reply:
Cortex-A is supported with my latest commit. This could break bisectability though.

Reply:
```
tests: kernel: interrupt: Enable nested interrupt test for Cortex-A

This commit enables nested interrupt test for the Cortex-A platforms
that use the ARM Generic Interrupt Controller (GIC).

Signed-off-by: Stephanos Ioannidis <root@stephanos.io>
```

The intention here is to enable nested interrupt testing for Cortex-A, which it does.

Reply:
I mean, the NOTE above seems wrong then.

Reply:
> I mean, the NOTE above seems wrong then.

You're right, it seems I forgot to update the comment. I will fix that.

Reply:
Fixed